app/views/static_pages/board.html.erb

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html >
+<html>
+<style type="text/css">
```

Comment:
Let's add this into a CSS assets file, likely for the board. It's best to keep everything in their little isolated niche so that it is easier to find things when it comes time for tweaks

Take a look at this article, which gets into the pipeline and all that - https://launchschool.com/blog/rails-asset-pipeline-best-practices